src/components/ebay-skeleton/components/ebay-skeleton-avatar/index.marko

```diff
@@ -3,7 +3,7 @@ import type { AttrClass } from "marko/tags-html";
 import type { WithNormalizedProps } from '../../../../global';
 
 static interface SkeletonAvatarInput extends Omit<Marko.Input<"div">, `on${string}`> {
-    as?: Marko.NativeTags;
+    as?: keyof Marko.NativeTags;
```

Comment:
What happened to cause these out of scope changes?

Reply:
I don't think it was a merge conflict, I added them in a recent commit because I didn't think this TS change warranted a whole PR. Mentioned [here](https://github.com/eBay/ebayui-core/pull/2289#issuecomment-2438374282).